internal/command/local/connect.go

```diff
@@ -109,25 +109,27 @@ func runConnect(cmd *cobra.Command, cfg *config.LocalConnect, args []string) err
 		GetCmd: func() *exec.Cmd {
 			var cmdToRun *exec.Cmd
 			if !cfg.Unpriveleged {
-				cmdToRun = exec.Command("sudo",
+				cmdToRun = exec.Command(
+					"sudo",
+					"-S",
 					"--preserve-env=SIGNADOT_LOCAL_CONNECT_INVOCATION_CONFIG",
-					os.Args[0], "locald")
+					os.Args[0],
+					"locald",
+				)
 			} else {
 				cmdToRun = exec.Command(os.Args[0], "locald")
 				cmdToRun.Env = append(cmdToRun.Env,
-					fmt.Sprintf("SIGNADOT_LOCAL_CONNECT_INVOCATION_CONFIG=%s", string(ciBytes)),
 					fmt.Sprintf("HOME=%s", ciConfig.UIDHome),
 					fmt.Sprintf("PATH=%s", ciConfig.UIDPath))
 			}
+			cmdToRun.SysProcAttr = &syscall.SysProcAttr{
```

Comment:
I would have thought this should only apply to the sudo case.  It works fine for me without setsid in the unprivileged case.  wdyt?